nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -61,13 +61,9 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           type: label-expression
           name: AVAILABLE_DIST
           values:
-            - centos6
```

Comment:
this section would list everything that you think the project is able to build. That is why this is named `AVAILABLE_DIST`. It allows for filtering the list from user input, e.g. 'just build in centos7'. You can remove centos6 if that will never ever be built (I think that may be the case here)